### PR TITLE
HBASE-25273 fix typo in StripeStoreFileManager java doc

### PR DESCRIPTION
**Pull request by empire**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 49s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m  7s |  master passed  |
| +1 :green_heart: |  spotbugs  |   2m  1s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 24s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m  3s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  17m 16s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   2m  7s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 15s |  The patch does not generate ASF License warnings.  |
|  |   |  38m 43s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2646 |
| Optional Tests | dupname asflicense spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux a92a26abe162 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f0c430aed2 |
| Max. process+thread count | 94 (vs. ulimit of 30000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  2s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 19s |  master passed  |
| +1 :green_heart: |  compile  |   1m  6s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 48s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 43s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m  3s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m  4s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m  4s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 44s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 40s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 138m 55s |  hbase-server in the patch passed.  |
|  |   | 167m 53s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2646 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux d63f736f654b 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f0c430aed2 |
| Default Java | AdoptOpenJDK-11.0.6+10 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/testReport/ |
| Max. process+thread count | 3925 (vs. ulimit of 30000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  8s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m  9s |  master passed  |
| +1 :green_heart: |  compile  |   0m 59s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 37s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 48s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 59s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 59s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 11s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 35s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 208m 13s |  hbase-server in the patch passed.  |
|  |   | 236m 38s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2646 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 1b876fab4050 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f0c430aed2 |
| Default Java | AdoptOpenJDK-1.8.0_232-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/testReport/ |
| Max. process+thread count | 3384 (vs. ulimit of 30000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2646/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
  * When L0 is compacted, it's split into the files corresponding to existing stripe boundaries,
  * that can thus be added to stripes.
  * When scan or get happens, it only has to read the files from the corresponding stripes.
- * See StripeCompationPolicy on how the stripes are determined; this class doesn't care.
+ * See StripeCompactionPolicy on how the stripes are determined; this class doesn't care.
```
Comment (HorizonNet) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java:59]: Also add a Javadoc link to the class?

**Review by HorizonNet — COMMENTED** [review]
Left a minor point.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java (hunk 1 above)
Comment (empire) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java:59]: @HorizonNet, I made a mistake and this pull request was closed and a [new pull request](https://github.com/apache/hbase/pull/2653) is opened.

Comment (empire) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java:59]: I've done your comment in that pull request.

**Pull request closed without merging**